### ISSUE TITLE
Page title should have consistent ending with  either "- Canada.ca" or "| RSAMS"

### ISSUE THREAD

**Issue by hydegeoff** [mmiron-HC/WST-RSAMS#157]
URL: https://rsams.dev.cloud.openplus.ca/en/user/
User Agent: Chrome v99
Language: Impacts EN and FR pages

Issue Description:  Consistency will help reduce user confusion and improve User experience with the screen reader.  All pages should  have a ending with either "- Canada.ca" or "| RSAMS".  All public facing should have an ending of  "- Canada.ca" and all Admin pages should be ending with "| RSAMS".

Note: This may impact other pages on the site and page title should be consistent with ending above.

These are problems that were found based on details above:
* The Dashboard page title should remove the username and have "| RSAMS" as shown below 
(this could be a security concerns well)
* The page immediately after clicking build should end with "| RSAMS" as shown below
* These help pages are specifically for admin and are internal pages only so they should end with "| RSAMS"
![Page TItles endings should be updated](https://user-images.githubusercontent.com/88147013/162063541-542f135d-ffe7-4510-971f-e51557530469.JPG)

Expected results:
Page titles should have consistent page ending in accordance with TBS guidelines.

**Comment by echouin** [comment]
Hi @hydegeoff  - note sure about this. The reason that the tab does not have Canada.ca is because it is a user's dashboard. It needs to be clear that this is the RSA CMS User's primary space to build and manage recall content. Does that make sense?

**Comment by hydegeoff** [comment]
@echouin, Yes, "RSA CMS" does make sense, if you want to end with this.  You may want to spell out the abbreviation unless you feel RSA CMS is well known.

**Comment by echouin** [comment]
@hydegeoff I actually think this is fine as it appears. As you can see from the Screenshot it says: User Dashboard | ghyde (your username). I think that this is perfectly fine to close as it will appear unique to all users and indicates that they are in the CMS and that they are using their account :)

**Comment by hydegeoff** [comment]
@echouin The page title is unique, because if I remember there is no other page that is named Dashboard, but the page title suppose to have site name in it as well, instead of "ghyde"

**Comment by echouin** [comment]
Hi @hydegeoff I'm not sure I understand the issue. The tab says: 'User Dashboard' and then shows the unique user, in this instance, your username. This is a back-end CMS tab, so I think its working fine and as intended, no?

For instance, on the Prod version of the website, the tab let's me know when I'm signed into the dashboard at a glance:
![image](https://user-images.githubusercontent.com/88099454/192277798-4ff24677-cb3a-49a6-83f7-b67116195eb7.png)
